iceoryx_binding_c/source/cpp2c_enum_translation.cpp

```diff
@@ -89,6 +95,8 @@ iox_ListenerResult ListenerResult(const iox::popo::ListenerError value)
         return ListenerResult_EVENT_ALREADY_ATTACHED;
     case ListenerError::LISTENER_FULL:
         return ListenerResult_LISTENER_FULL;
+    case ListenerError::EMPTY_INVALIDATION_CALLBACK:
```

Comment:
Similar kind of non existent callback error but worded differently. Consistency?